README.md

```diff
@@ -1,21 +1,66 @@
 # ARMO cluster components
 ARMO Vulnerability Scanning
 
+![Version: 1.7.7](https://img.shields.io/badge/Version-1.7.7-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.7.7](https://img.shields.io/badge/AppVersion-v1.7.7-informational?style=flat-square)
 
-# Installing ARMO cluster components in a Kubernetes cluster Using Helm:
-=============================================================
+## [Docs](https://hub.armo.cloud/docs/installation-of-armo-in-cluster)
+
+## Installing ARMO cluster components in a Kubernetes cluster Using Helm:
 
 1. Add the Vulnerability Scanning Helm Repo
 ```
 helm repo add armo https://armosec.github.io/armo-helm/
 ```
 
-2. Get Your Account ID from ARMO
```

Comment:
@dwertent  why did you removed how to get account id?

Reply:
@dwertent  i would keep it in readme.

Reply:
ok